pkg/csi/cinder/controllerserver.go

```diff
@@ -32,19 +32,27 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/types/known/timestamppb"
+	"k8s.io/klog/v2"
 
 	sharedcsi "k8s.io/cloud-provider-openstack/pkg/csi"
 	"k8s.io/cloud-provider-openstack/pkg/csi/cinder/openstack"
 	"k8s.io/cloud-provider-openstack/pkg/util"
 	cpoerrors "k8s.io/cloud-provider-openstack/pkg/util/errors"
-	"k8s.io/klog/v2"
 )
 
 type controllerServer struct {
 	Driver *Driver
 	Clouds map[string]openstack.IOpenStack
 }
 
+type cloudsStartingToken struct {
```

Comment:
I also wonder whether it's better to use `<token>:<cloud>` format and use `SplitN` along with `Join` to avoid useless json encoding/decoding.

Reply:
I will provide a commit that changes the token to the colon-separated one. We don't care about any kind of backwards-compatibility here so it should be straightforward to change